lib/_http_outgoing.js

```diff
@@ -283,17 +285,26 @@ OutgoingMessage.prototype._send = function _send(data, encoding, callback) {
 OutgoingMessage.prototype._writeRaw = _writeRaw;
 function _writeRaw(data, encoding, callback) {
   const conn = this.socket;
-  if (conn && conn.destroyed) {
-    // The socket was destroyed. If we're still trying to write to it,
-    // then we haven't gotten the 'close' event yet.
-    return false;
-  }
 
   if (typeof encoding === 'function') {
     callback = encoding;
     encoding = null;
   }
 
+  if (conn && conn.destroyed) {
+    const err = new ERR_STREAM_DESTROYED('write');
+    if (callback) {
+      callback(err);
+    }
+    if (this.listenerCount('error') > 0 && this[kErrored] === undefined) {
```

Comment:
What's the reason for the listener count check? We don't do this for streams?

Reply:
nit, `!this[kErrored]`

Reply:
The reason is not to break a lot of tests, and keep this semver-minor.

Reply:
Could you add a comment for that at least? Then maybe we can "fix" it in a semver-major in the future?